inspirehep/modules/theme/views.py

```diff
@@ -646,6 +646,10 @@ def ajax_experiments_people():
         }
     )
 
+@blueprint.route('/account/settings/linkedaccounts/', methods=['GET'])
```

Comment:
Sorry, shouldn't we change the url `invenio-oauthclient` redirects to, rather than adding another redirect?